renovate.json

```diff
@@ -10,6 +10,7 @@
   },
   "automergeType": "branch",
   "rebaseWhen": "conflicted",
+  "minimumReleaseAge": "7 days",
```

Comment:
just as info: this is for everything, not just npm

Reply:
Yes, thanks for bringing it up. This is more or less on purpose. Even though the recent breaches have been on npm, other package managers and registries are not immune to such attacks.